### PR TITLE
Use Title Case and backtick for: Mix, rebar, and others

### PR DESCRIPTION
**Pull request by eksperimental**
Properly use Title Case for Mix, or backticks when it's refering
to the `mix` command

Since rebar name is in lowercase, I only added backticks when it was
refering to the `rebar` command itself.

I consider commands to be surrounded by backticks when they are used as nouns,
such as, or literally refered as commands,
but not when refered as adjectives, such as "a rebar dependency", "a make project"

I have ignored the test names when doing these changes.

Also fixes for other commands such as: `git`, `make`

when given in message such as exceptions, I only surround commands in backticks,
and arguments/values in single-quotes.

The following commands were used to find mentions on running a command that is not using backticks:

    commands="elixir|elixirc|mix|iex|ex_doc|git|make|rebar|dialyzer|erl|rm|cd|mkdir|rmdir|ln|ls|pwd"
    actions="runs?|running|executes?|executing|types?|typing|enters?|entering|calls?|calling"
    ag '(?i)('${actions}')(?-i)\b[^`\r\n/]+[\ \t]+(?!(`))('${commands}')'
    ag '(?-i)\b(?!(`))('${commands}')[\ \t]+[^`\r\n/]+(?i)(commands?)'

and this one to find where the commands were mentioned

    #commands="erlang|elixir|eex|iex|ex_unit|ex_doc|logger|elixirc|git|make|rebar|dialyzer|erl|rm|cd|mkdir|rmdir|ln|ls|pwd"
    commands="mix|elixirc|git|make|rebar|dialyzer|erl|rm|cd|mkdir|rmdir|ln|ls|pwd"
    ag -s '(?<!(\`))(?<!(\.))(?<!(/))(?<!(:))(?<!(_))\b('${commands}')\b(?!(:))(?!(\?))(?!(_))(?!(-))(?!(\.))(?!(\`))(?!(/))(?!(>))' \
    --ignore "*.erl" --ignore "*.yrl" --ignore "*.src"

### CODE REVIEW

**Comment by eksperimental** [comment]
@josevalim this is changed a few messages, and maybe too much for 1.1.0, so if you feel it's better to leave it for 1.2, just put it on hold until then.

**Comment by josevalim** [comment]
Awesome! Can we please use double quotes in the places we used single quotes?

File: lib/mix/lib/mix/tasks/archive.build.ex [hunk 1]
```diff
@@ -68,12 +68,12 @@ defmodule Mix.Tasks.Archive.Build do
     end
 
     unless File.dir?(source) do
-      Mix.raise "Expected archive source #{inspect source} to be a directory"
+      Mix.raise "Expected archive source '#{inspect source}' to be a directory"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/archive.build.ex]: `inspect source` will already surround it by double quotes.

File: lib/mix/lib/mix/tasks/archive.install.ex [hunk 2]
```diff
@@ -38,7 +38,7 @@ defmodule Mix.Tasks.Archive.Install do
 
       case Path.extname(path) do
         ".ez" -> install_archive(src, opts)
-        _     -> Mix.raise "mix archive.install doesn't know how to install #{path}"
+        _     -> Mix.raise "`mix archive.install` doesn't know how to install '#{path}'"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/archive.install.ex]: Let's not use single quotes. The best here is to use `#{inspect path}` which also hands escape.

File: lib/mix/lib/mix/tasks/archive.install.ex [hunk 3]
```diff
@@ -100,13 +100,13 @@ defmodule Mix.Tasks.Archive.Install do
...
   defp should_install?(src, []) do
-    Mix.shell.yes?("Are you sure you want to install archive #{src}?")
+    Mix.shell.yes?("Are you sure you want to install archive '#{src}'?")
   end
 
   defp should_install?(_src, previous_files) do
     files = Enum.map_join(previous_files, ", ", &Path.basename/1)
 
-    Mix.shell.yes?("Found existing archive(s): #{files}.\n" <>
+    Mix.shell.yes?("Found existing archive(s): '#{files}'.\n" <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/archive.install.ex]: This is going to show a list of files, so the quotes do not make sense.

File: lib/mix/lib/mix/tasks/deps.clean.ex [hunk 4]
```diff
@@ -40,10 +40,10 @@ defmodule Mix.Tasks.Deps.Clean do
       apps != [] ->
         do_clean(apps, build, deps)
       true ->
-        Mix.raise "mix deps.clean expects dependencies as arguments or " <>
+        Mix.raise "`mix deps.clean` expects dependencies as arguments or " <>
                   "a flag indicating which dependencies to clean. " <>
-                  "The --all option will clean all dependencies while " <>
-                  "the --unused option cleans unused dependencies"
+                  "The '--all' option will clean all dependencies while " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.clean.ex]: No need to surround the switches by quotes.

File: lib/mix/lib/mix/tasks/archive.build.ex (hunk 1 above)
Comment (eksperimental) [inline lib/mix/lib/mix/tasks/archive.build.ex]: yes, was aware of this. I just overlooked at it.

**Comment by josevalim** [comment]
I am a little torn with some of those changes. I have typically avoided adding backticks to error messages because we are often fine without them. For example, things like `"--all"` just add verbosity, we can definitely read `--all` without ambiguity.

**Comment by josevalim** [comment]
In other words, while we need to consistently format those in the documentation, in error messages we should prefer readability instead of following a strict syntax that won't be ever formatted in error messages.

File: lib/mix/lib/mix/rebar.ex [hunk 5]
```diff
@@ -140,8 +140,8 @@ defmodule Mix.Rebar do
         config
       {:error, error} ->
         reason = :file.format_error(error)
-        Mix.shell.error("Error evaluating rebar config script #{script_path}: #{reason}")
-        Mix.shell.error("You may solve this issue by adding rebar as a dependency to your project")
+        Mix.shell.error("Error evaluating rebar config script '#{script_path}': #{reason}")
+        Mix.shell.error("You may solve this issue by adding `rebar` as a dependency to your project")
```
Comment (eksperimental) [inline lib/mix/lib/mix/rebar.ex]: maybe we don't need quotes here?

**Comment by josevalim** [comment]
So I would do the following changes:

1. Do not use single quotes anywhere. We should favor double quotes everywhere, to avoid confusion

2. If you want to format something in error messages, use `inspect`. For example, if you want to show the dependency name and that is an atom, instead of `the dependency "foo"`, let's show `the dependency :foo`. Less noise and may click better

3. Similarly, if you want to show something with double quotes, call inspect, as it handles escaping as well as the quotes

Thoughts?

File: lib/mix/lib/mix/rebar.ex [hunk 6]
```diff
@@ -140,8 +140,8 @@ defmodule Mix.Rebar do
         config
       {:error, error} ->
         reason = :file.format_error(error)
-        Mix.shell.error("Error evaluating rebar config script #{script_path}: #{reason}")
-        Mix.shell.error("You may solve this issue by adding rebar as a dependency to your project")
+        Mix.shell.error("Error evaluating rebar config script '#{script_path}': #{reason}")
```
Comment (eksperimental) [inline lib/mix/lib/mix/rebar.ex]: maybe we don't need backticks here,
and for script, i can use inspect

**Comment by eksperimental** [comment]
that is perfect @josevalim 
very concise,
i would only add the rules about the switches, no need to use quotes, when used double hyphen., ...what about only one hyphen?

**Comment by josevalim** [comment]
> what about only one hyphen?

Maybe we can make it explicit in the text. "... give the switch -o when choosing ..."

?

**Comment by ericmj** [comment]
I am :-1: on surrounding all instances of mix, rebar, dialyzer, git, etc. with backticks. In an example of how the command is called it should be backticked but when generally talking about the application/tool there should not be backticks since it's just a name.

**Comment by josevalim** [comment]
@ericmj they were not all surrounded. If they were being referred as a name, Titlecase was used. Backticks were used only when referring to the command line tool.

**Comment by eksperimental** [comment]
but @ericmj could be right with some cases,
because rebar, git, doens't use the title case.

Here's an example of what i tried to do:

> Whenever a doctest starts with "#Name<", `doctest` will perform a string

you can see, the first "doctest" is not referent to the command, so I doesn't surround it by backticks.

I will update the changes, submit them as a new commit (so easier to review), and if you guys wanna comment one what should not use backtick, I can change it.

**Comment by josevalim** [comment]
Right. If we are referring only to git and not the command, then no backticks. :)

**Comment by eksperimental** [comment]
all done.
I left it in two commits so it's easier to see what was changed now.
I will rebase once finished,
comments are welcome.

File: lib/ex_unit/lib/ex_unit.ex [hunk 7]
```diff
@@ -118,7 +118,7 @@ defmodule ExUnit do
         1. per test by setting "@tag timeout: x"
         2. per case by setting "@moduletag timeout: x"
         3. globally via "ExUnit.start(timeout: x)" configuration
-        4. or set it to infinity per run by calling "mix test --trace"
+        4. or set it to infinity per run by calling `mix test --trace`
```
Comment (josevalim) [inline lib/ex_unit/lib/ex_unit.ex]: no backticks in error messages. they are not markdown :)

File: lib/mix/lib/mix/cli.ex [hunk 8]
```diff
@@ -35,8 +35,8 @@ defmodule Mix.CLI do
   end
 
   defp get_task(["-" <> _|_]) do
-    Mix.shell.error "** (Mix) Cannot implicitly pass flags to default mix task, " <>
-                    "please invoke instead: mix #{Mix.Project.config[:default_task]}"
+    Mix.shell.error "** (Mix) Cannot implicitly pass flags to default Mix task, " <>
+                    "please invoke instead `mix #{Mix.Project.config[:default_task]}`"
```
Comment (josevalim) [inline lib/mix/lib/mix/cli.ex]: no backticks in error messages. they are not markdown :)

File: lib/mix/lib/mix/rebar.ex [hunk 9]
```diff
@@ -140,8 +140,8 @@ defmodule Mix.Rebar do
         config
       {:error, error} ->
         reason = :file.format_error(error)
-        Mix.shell.error("Error evaluating rebar config script #{script_path}: #{reason}")
-        Mix.shell.error("You may solve this issue by adding rebar as a dependency to your project")
+        Mix.shell.error("Error evaluating rebar config script #{inspect script_path}: #{reason}")
+        Mix.shell.error("You may solve this issue by adding :rebar as a dependency to your project")
```
Comment (josevalim) [inline lib/mix/lib/mix/rebar.ex]: I think this whole line today it is wrong. lets just remove it

File: lib/mix/lib/mix/rebar.ex [hunk 10]
```diff
@@ -157,7 +157,7 @@ defmodule Mix.Rebar do
   defp wrap_cmd(nil), do: nil
   defp wrap_cmd(rebar) do
     if match?({:win32, _}, :os.type) and not String.ends_with?(rebar, ".cmd") do
-      "escript.exe \"#{rebar}\""
+      "escript.exe #{inspect rebar}"
```
Comment (josevalim) [inline lib/mix/lib/mix/rebar.ex]: this will be used by System.cmd, we shouldn't use inspect

File: lib/mix/lib/mix/tasks/archive.build.ex [hunk 11]
```diff
@@ -73,7 +73,7 @@ defmodule Mix.Tasks.Archive.Build do
 
     Mix.Archive.create(source, target)
 
-    Mix.shell.info "Generated archive #{target} with MIX_ENV=#{Mix.env}"
+    Mix.shell.info "Generated archive #{inspect target} with \"MIX_ENV=#{Mix.env}\""
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/archive.build.ex]: :+1: on inspect, :-1: on the quotes

File: lib/mix/lib/mix/tasks/archive.install.ex [hunk 12]
```diff
@@ -38,7 +38,7 @@ defmodule Mix.Tasks.Archive.Install do
 
       case Path.extname(path) do
         ".ez" -> install_archive(src, opts)
-        _     -> Mix.raise "mix archive.install doesn't know how to install #{path}"
+        _     -> Mix.raise "`mix archive.install` doesn't know how to install #{inspect path}"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/archive.install.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.clean.ex [hunk 13]
```diff
@@ -40,7 +40,7 @@ defmodule Mix.Tasks.Deps.Clean do
       apps != [] ->
         do_clean(apps, build, deps)
       true ->
-        Mix.raise "mix deps.clean expects dependencies as arguments or " <>
+        Mix.raise "`mix deps.clean` expects dependencies as arguments or " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.clean.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 14]
```diff
@@ -105,7 +105,7 @@ defmodule Mix.Tasks.Deps.Compile do
         kind, reason ->
           stacktrace = System.stacktrace
           app = dep.app
-          Mix.shell.error "could not compile dependency #{app}, mix compile failed. " <>
+          Mix.shell.error "could not compile dependency #{inspect app}, `mix compile` failed. " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 15]
```diff
@@ -125,16 +125,16 @@ defmodule Mix.Tasks.Deps.Compile do
 
   defp handle_rebar_not_found(app) do
     shell = Mix.shell
-    shell.info "Could not find rebar, which is needed to build dependency #{inspect app}"
-    shell.info "I can install a local copy which is just used by mix"
+    shell.info "Could not find `rebar`, which is needed to build dependency #{inspect String.to_atom(app)}"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 16]
```diff
@@ -125,16 +125,16 @@ defmodule Mix.Tasks.Deps.Compile do
...
+    shell.info "I can install a local copy which is just used by Mix"
 
     unless shell.yes?("Shall I install rebar?") do
-      Mix.raise "Could not find rebar to compile " <>
-        "dependency #{app}, please ensure rebar is available"
+      Mix.raise "Could not find `rebar` to compile " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 17]
```diff
@@ -125,16 +125,16 @@ defmodule Mix.Tasks.Deps.Compile do
...
+        "dependency #{inspect String.to_atom(app)}, please ensure `rebar` is available"
     end
 
     (Mix.Tasks.Local.Rebar.run([]) && Mix.Rebar.local_rebar_cmd) ||
-      Mix.raise "rebar installation failed"
+      Mix.raise "`rebar` installation failed"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 18]
```diff
@@ -158,8 +158,8 @@ defmodule Mix.Tasks.Deps.Compile do
     Mix.Dep.in_dependency dep, fn _ ->
       env = [{"ERL_LIBS", Path.join(config[:build_path], "lib")}]
       if Mix.shell.cmd("#{command} #{extra}", print_app: print_app?, env: env) != 0 do
-        Mix.raise "Could not compile dependency #{app}, #{command} command failed. " <>
-          "If you want to recompile this dependency, please run: mix deps.compile #{app}"
+        Mix.raise "Could not compile dependency #{inspect String.to_atom(app)}, `#{command}` command failed. " <>
+          "If you want to recompile this dependency, please run: `mix deps.compile #{app}`"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 19]
```diff
@@ -158,8 +158,8 @@ defmodule Mix.Tasks.Deps.Compile do
     Mix.Dep.in_dependency dep, fn _ ->
       env = [{"ERL_LIBS", Path.join(config[:build_path], "lib")}]
       if Mix.shell.cmd("#{command} #{extra}", print_app: print_app?, env: env) != 0 do
-        Mix.raise "Could not compile dependency #{app}, #{command} command failed. " <>
-          "If you want to recompile this dependency, please run: mix deps.compile #{app}"
+        Mix.raise "Could not compile dependency #{inspect String.to_atom(app)}, `#{command}` command failed. " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.compile.ex [hunk 20]
```diff
@@ -125,16 +125,16 @@ defmodule Mix.Tasks.Deps.Compile do
...
-    shell.info "Could not find rebar, which is needed to build dependency #{inspect app}"
-    shell.info "I can install a local copy which is just used by mix"
+    shell.info "Could not find `rebar`, which is needed to build dependency #{inspect String.to_atom(app)}"
+    shell.info "I can install a local copy which is just used by Mix"
 
     unless shell.yes?("Shall I install rebar?") do
-      Mix.raise "Could not find rebar to compile " <>
-        "dependency #{app}, please ensure rebar is available"
+      Mix.raise "Could not find `rebar` to compile " <>
+        "dependency #{inspect String.to_atom(app)}, please ensure `rebar` is available"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.compile.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.ex [hunk 21]
```diff
@@ -40,7 +40,7 @@ defmodule Mix.Tasks.Deps do
 
   Path and in umbrella dependencies are automatically recompiled by
   the parent project whenever they change. While fetchable dependencies
-  like git are recompiled only when fetched/updated.
+  like `:git` are recompiled only when fetched/updated.
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.ex]: "like the ones using :git"

File: lib/mix/lib/mix/tasks/deps.unlock.ex [hunk 22]
```diff
@@ -44,7 +44,7 @@ defmodule Mix.Tasks.Deps.Unlock do
         Mix.Dep.Lock.write(lock)
 
       true ->
-        Mix.raise "mix deps.unlock expects dependencies as arguments or " <>
+        Mix.raise "`mix deps.unlock` expects dependencies as arguments or " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.unlock.ex]: no backticks

File: lib/mix/lib/mix/tasks/deps.update.ex [hunk 23]
```diff
@@ -31,7 +31,7 @@ defmodule Mix.Tasks.Deps.Update do
         {old, new} = Dict.split(Mix.Dep.Lock.read, to_app_names(rest))
         Mix.Dep.Fetcher.by_name(rest, old, new, fetch_opts)
       true ->
-        Mix.raise "mix deps.update expects dependencies as arguments or " <>
+        Mix.raise "`mix deps.update` expects dependencies as arguments or " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/deps.update.ex]: no backticks

File: lib/mix/lib/mix/tasks/help.ex [hunk 24]
```diff
@@ -147,12 +147,12 @@ defmodule Mix.Tasks.Help do
 
   defp display_default_task_doc(max) do
     Mix.shell.info format_task("mix", max,
-                    "Run the default task (current: mix #{Mix.Project.config[:default_task]})")
+                    "Runs the default task (current: `mix #{Mix.Project.config[:default_task]}`)")
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/help.ex]: no backticks

File: lib/mix/lib/mix/tasks/iex.ex [hunk 25]
```diff
@@ -7,6 +7,6 @@ defmodule Mix.Tasks.Iex do
 
   @spec run(OptionParser.argv) :: no_return
   def run(_) do
-    Mix.raise "To use IEx with Mix, please run: iex -S mix"
+    Mix.raise "To use IEx with Mix, please run: `iex -S mix`"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/iex.ex]: no backticks

File: lib/mix/lib/mix/tasks/local.rebar.ex [hunk 26]
```diff
@@ -5,23 +5,24 @@ defmodule Mix.Tasks.Local.Rebar do
   @rebar_list_url     @rebar_s3 <> "/installs/rebar-1.x.csv"
   @rebar_escript_url  @rebar_s3 <> "/installs/[VERSION]/rebar"
 
-  @shortdoc  "Installs rebar locally"
+  @shortdoc  "Installs `rebar` locally"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/local.rebar.ex]: no backticks. this is used without formating on mix help

File: lib/mix/lib/mix/tasks/local.rebar.ex [hunk 27]
```diff
@@ -52,7 +53,7 @@ defmodule Mix.Tasks.Local.Rebar do
           Mix.raise """
           #{message}
 
-          Could not fetch rebar at:
+          Could not fetch `rebar` at:
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/local.rebar.ex]: no backticks

File: lib/mix/lib/mix/tasks/new.ex [hunk 28]
```diff
@@ -98,8 +98,8 @@ defmodule Mix.Tasks.New do
 
     Mix.shell.info """
 
-    Your mix project was created successfully.
-    You can use mix to compile it, test it, and more:
+    Your Mix project was created successfully.
+    You can use `mix` to compile it, test it, and more:
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/new.ex]: no backticks

File: lib/mix/lib/mix/tasks/test.ex [hunk 29]
```diff
@@ -141,8 +141,8 @@ defmodule Mix.Tasks.Test do
     {opts, files, _} = OptionParser.parse(args, switches: @switches)
 
     unless System.get_env("MIX_ENV") || Mix.env == :test do
-      Mix.raise "mix test is running on environment #{Mix.env}. If you are " <>
-                                "running tests along another task, please set MIX_ENV explicitly"
+      Mix.raise "`mix test` is running on environment \"#{Mix.env}\". If you are " <>
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/test.ex]: no backticks

File: lib/mix/test/mix/tasks/deps.git_test.exs [hunk 30]
```diff
@@ -158,7 +158,7 @@ defmodule Mix.Tasks.DepsGitTest do
       assert File.exists?("deps/git_repo/.fetch")
       ensure_touched("deps/git_repo/.fetch") # Ensure timestamp differs
 
-      # mix deps.compile is required...
+      # `mix deps.compile` is required...
```
Comment (josevalim) [inline lib/mix/test/mix/tasks/deps.git_test.exs]: no backticks

File: lib/mix/test/mix/tasks/deps_test.exs [hunk 31]
```diff
@@ -575,7 +575,7 @@ defmodule Mix.Tasks.DepsTest do
       File.mkdir_p!("_build/dev/lib/git_repo")
       File.mkdir_p!("_build/test/lib/git_repo")
 
-      message = "mix deps.clean expects dependencies as arguments or " <>
+      message = "`mix deps.clean` expects dependencies as arguments or " <>
```
Comment (josevalim) [inline lib/mix/test/mix/tasks/deps_test.exs]: no backticks

**Comment by josevalim** [comment]
I have added some comments. Please don't use backticks unless inside documentation. Don't use them for error messages, warnings nor code comments. :)

**Comment by eksperimental** [comment]
so sorry @josevalim , I thought it was ok to use backticks in the error messages.
So your proposal is to replace them with double-quotes? or remove them completely?

**Comment by eksperimental** [comment]
my concern is when we have commands, like
> Mix.raise "\`mix archive.install\` doesn't know how to install #{inspect path}"

and not when we mention a command (we can get away with any quotes whatsoever),
> You can use \`mix\` to compile it, test it, and more:

**Comment by josevalim** [comment]
I am fine with removing them or using double quotes. My focus here is not
consistency, but readability and usability.

-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by eksperimental** [comment]
@josevalim finally done.

All references to backticks have been replaced with double quotes everywhere the
code is not interpreted as Markdown, (ie. anywhere outside documentation and
markdown files).

If you can review the changes involving the use of inspect in the libraries:
* lib/elixir/lib/exception.ex
* lib/elixir/lib/inspect/algebra.ex
and their tests (which had to be adapted a bit):
* lib/elixir/test/elixir/inspect_test.exs
* lib/ex_unit/test/ex_unit/formatter_test.exs
* lib/elixir/test/elixir/exception_test.exs

looking forward to hearing your input, or anybody else's

File: lib/elixir/lib/module.ex [hunk 32]
```diff
@@ -12,7 +12,7 @@ defmodule Module do
   ## Module attributes
 
   Each module can be decorated with one or more attributes. The following ones
-  are currently defined by Elixir:
+  are currently defined by Elixir:`
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: oops :)

File: lib/mix/lib/mix/rebar.ex [hunk 33]
```diff
@@ -157,7 +156,7 @@ defmodule Mix.Rebar do
   defp wrap_cmd(nil), do: nil
   defp wrap_cmd(rebar) do
     if match?({:win32, _}, :os.type) and not String.ends_with?(rebar, ".cmd") do
-      "escript.exe \"#{rebar}\""
+       "escript.exe \"#{rebar}\""
```
Comment (josevalim) [inline lib/mix/lib/mix/rebar.ex]: oops. unecessrary extra space :)

**Comment by josevalim** [comment]
This looks great, everything is much more consistent now. I have added just two tiny comments and we are ready to :shipit:!

File: lib/elixir/lib/module.ex (hunk 32 above)
Comment (eksperimental) [inline lib/elixir/lib/module.ex]: good eye. I spotted before . I must have put it back to test the regex! thanks!

**Comment by eksperimental** [comment]
I'm happy you like it,
I have corrected locally those two little issues..
Should I squash the first 3 commits?

**Comment by josevalim** [comment]
Yes, as you seem fit. :)

**Comment by eksperimental** [comment]
there you go....!!!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**